google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/LoadConfiguration.java

```diff
@@ -92,16 +92,6 @@
      */
     Builder setIgnoreUnknownValues(Boolean ignoreUnknownValues);
 
-
-    /**
-     * Sets which entity properties to load into BigQuery from a Cloud Datastore backup. This field
-     * is only used if the source format is set to {@code DATASTORE_BACKUP}. Property names are case
-     * sensitive and must be top-level properties. If no properties are specified, BigQuery loads
-     * all properties. If any named property isn't found in the Cloud Datastore backup, an invalid
-     * error is returned in the job result.
-     */
-    Builder setProjectionFields(List<String> projectionFields);
```

Comment:
DataStoreBackupOptions is a subclass of FormatOptions so it is set through `setFormatOptions` method